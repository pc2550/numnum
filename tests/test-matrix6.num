
int main(){
	float [2][1] a;
	float [2][1] b;
	float [2][1] c;

    a[0][0] = 2.0;
    a[1][0] = 4.0; 
    b[0][0] = 3.0;
    b[1][0] = 3.0;
    c[0][0] = 1.0;
    c[1][0] = 1.0;

    el_sub(a, b, c);

    printfl(c[0][0]);
    printfl(c[1][0]);

    el_add(a, b, c);

    printfl(c[0][0]);
    printfl(c[1][0]);

    el_mul(a, b, c);

    printfl(c[0][0]);
    printfl(c[1][0]);

    el_div(a, b, c);

    printfl(c[0][0]);
    printfl(c[1][0]);

	return 0;
}
