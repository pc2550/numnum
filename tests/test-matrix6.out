-1.000000
1.000000
5.000000
7.000000
6.000000
12.000000
0.666667
1.333333
